plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtDomainXMLParser.java

```diff
@@ -203,6 +202,12 @@ public boolean parseDomainXML(String domXML) {
                 String model = getAttrValue("model", "type", nic);
                 String slot = StringUtils.removeStart(getAttrValue("address", "slot", nic), "0x");
 
+                String mtuAttrValue = getAttrValue("mtu", "size", nic);
+                int mtu = 1500;
+                if (mtuAttrValue != null) {
+                    mtu = Integer.parseInt(mtuAttrValue);
+                }
```

Comment:
So we always set the MTU to 1500 if it's not set. Shouldn't we just ignore the MTU if it's not set?

Leave it to the libvirt default unless somebody specifies it

Reply:
Correct. I change it the an **Integer** with default null value.